### PR TITLE
feat: Add Airbrake Netlify plugin

### PR DESCRIPTION
**Pull request by bharathvaj-ganesan**
Thanks for contributing the Netlify plugins directory!

**Are you adding a plugin or updating one?**

- [x] Adding a plugin
- [ ] Updating a plugin

**Have you completed the following?**

- [x] Read and followed the [plugin author guidelines](https://github.com/netlify/plugins/blob/main/docs/guidelines.md).
- [x] Included all [required fields](https://github.com/netlify/plugins/blob/main/docs/CONTRIBUTING.md#required-fields) in your entry.
- [x] Tested the plugin [locally](https://docs.netlify.com/cli/get-started/#run-builds-locally) and [on Netlify](https://docs.netlify.com/configure-builds/build-plugins/#install-a-plugin), using the plugin version stated in your entry.

**Test plan**
https://app.netlify.com/sites/pedantic-tesla-a2d422/deploys/60978ba180c8561836aa04ee

### CODE REVIEW

**Review by ehmicky — CHANGES_REQUESTED** [review]
Thanks for this new plugin @bharathvaj-ganesan! :tada: 

This looks great. I just had one small comment about the plugin description, please let me know what you think.

File: site/plugins.json [hunk 1]
```diff
@@ -482,5 +482,13 @@
     "package": "netlify-plugin-ghost-inspector",
     "repo": "https://github.com/ghost-inspector/netlify-plugin",
     "version": "1.0.1"
+  },
+  {
+    "author": "bharathvaj-ganesan",
+    "description": "The Airbrake Netlify build plugin automatically notifies Airbrake of new releases being deployed to your site",
```
[the remark this answers is not in the document]
Comment (bharathvaj-ganesan) [reply site/plugins.json:488]: @ehmicky Thanks for the feedback. I feel _"Send information about new deploys to Airbrake, error monitoring and performance insights for your entire app stack"_ should be the right one but the plugin currently focuses only on notifying airbrake about the deployments made. For now lets keep it as _"Automatically notifies Airbrake of new site deploys"_ as per your suggestion.

File: site/plugins.json [hunk 2]
```diff
@@ -431,7 +431,9 @@
     "compatibility": [
       {
         "version": "1.1.5",
-        "siteDependencies": { "next": "<10.0.6" }
+        "siteDependencies": {
+          "next": "<10.0.6"
+        }
```
Comment (ehmicky) [inline site/plugins.json:436]: There are two unrelated changes (this one and the final newline), would it be ok to undo them? Thanks!

File: site/plugins.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ehmicky) [reply site/plugins.json]: This sounds great!

File: site/plugins.json (hunk 2 above)
Comment (bharathvaj-ganesan) [reply site/plugins.json]: @ehmicky Cool. I have reverted these changes.

**Review by ehmicky — APPROVED** [review]

File: site/plugins.json (hunk 2 above)
Comment (ehmicky) [reply site/plugins.json]: Thanks!
